tests/basic/src/main/java/mil/emp3/dev_test_sdk/MainActivity.java

```diff
@@ -538,9 +534,9 @@ protected IGeoPosition getRandomCoordinate() {
         return oPosList;
     }
 
-    private java.util.List<IGeoPosition> getPositions(int iCount, double dAltitude) {
+    private List<IGeoPosition> getPositions(int iCount, double dAltitude) {
```

Comment:
You can't make this:

`private List<IGeoPosition> getPositions(final int iCount, final double dAltitude) {` ?